libc/spec/stdc.td

```diff
@@ -934,6 +934,13 @@ def StdC : StandardSpec<"stdc"> {
                ArgSpec<ConstCharRestrictedPtr>,
                ArgSpec<VarArgType>]
           >,
+          FunctionSpec<
```

Comment:
since we're moving to new headergen soon, you also need to add these functions to the config for that. Here's how:

in `libc/newhdrgen/yaml/stdio.yaml` add two new items to the list of functions:
```
  - name: asprintf
    standards: 
      - GNUExtensions
    return_type: int
    arguments:
      - type: char **__restrict
      - type: const char *__restrict
      - type: ...
  - name: vasprintf
    standards: 
      - GNUExtensions
    return_type: int
    arguments:
      - type: char **__restrict
      - type: const char *__restrict
      - type: va_list
```